### DISCUSSION TITLE
About event-camera support

### DISCUSSION THREAD

**Discussion by ahmetakman** [lava-nc/lava#214] [General]
Hello, 
I was curious about the real-time input capabilities of the library and I have not found any example that directly utilizes a real-time input. Will there be any plans of adding an abstraction layer or packet to make the lava processes easily accept event-based input (e. g. camera)? 

Sincerely,

**Comment by awintel** [comment]
Yes, we are planning to enable support for RGB, DVS cameras and perhaps RealSense in Q2 of 2022.

This will be made available as Lava Procs and corresponding ProcModels.

Andreas

**Reply by GBrayshaw** [reply]
Hi, Is there any update on DVS support?

George

**Comment by stevenabreu7** [comment]
Hi! Is there an update on DVS support? Also an interface for using the event data generated by the DVS (offline) would be great. Happy to contribute.

**Comment by mathisrichter** [comment]
Hi everyone

We have a prototype of a Lava Process that streams from an aedat file, using the `dv` library. We used this for the motion tracking demo I showed in the INRC workshop - in case some of you saw that. We are currently working on making that available. This Process can be used to stream from recorded DVS data to other Lava Processes, for instance a Dense Process on a Loihi neurocore (via a CProcess).
We may be able to make this Process available in the next Lava release, end of 2022.

In a second step, we will make a Process available that can stream directly from a DVS camera.
